### DISCUSSION TITLE
Please merge pull requests

### DISCUSSION THREAD

**Discussion by lukehutch** [serverpod/serverpod#1003] [General]
**Describe the bug**  (this is a "process bug" not a software bug...)

Is there any chance that the speed at which pull requests are approved (or even rejected / modified) could be increased by a bit, please? I have submitted a few pull requests. (I thought I submitted several more, but it looks like those didn't make it back to the main project from my fork... I'll work on that.) I have a bunch more changes I need to submit, but the approval process is slow.

Meanwhile, upstream Serverpod is under active development, so keeping my fork up to date vs. the main branch is becoming more and more complicated. Furthermore, a number of the changes I need are interdependent, so they can't easily be submitted as changes on separate feature branches.

Expedited handling of pull requests will increase the likelihood you will get repeat contributions coming in!

**Comment by vlidholt** [comment]
We're working hard to stay on top of everything, and typically review pull request within a few days. Some PRs take longer time, especially if they need manual testing and extra documentation. The Facebook sign-in is one example.

If the PRs are missing tests or are otherwise incomplete, the Serverpod tean will need to find time to write the tests to include the PRs. I'm sure we can get better at being faster in providing feedback though.

**Reply by lukehutch** [reply]
All good, thanks... I'm just trying to be as helpful as I can be to the project, but I keep hitting roadblocks along the way.

Is there a Discord server or something that could get faster turnaround on quick blocker questions?

I'm curious, what company is paying for the development of Serverpod? Or is it all volunteer work?

**Reply by vlidholt** [reply]
All feedback and work on Serverpod is very much appreciated! Thank you!! :)

Serverpod is an independent company, we're venture funded and have a small team of full time employees. The business plan is to eventually provide hosting for Serverpod projects.

**Reply by lukehutch** [reply]
Awesome, that gives me more confidence that I can rely on Serverpod development into the future.

Since you have a some fulltime employees working on Serverpod, it would be awesome if you could set up a Discord channel. GitHub is sometimes just too asynchronous, and I can be blocked on some stupid little question for 3 days, which would take one of you 5 minutes to solve...

What timezone are you and your employees in?

**Reply by vlidholt** [reply]
We may be doing a Slack pretty soon. Serverpod is based in Stockholm.
